HW/feature_eng.py

```diff
@@ -0,0 +1,61 @@
+import sys
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+# user input to receive dataset
+# https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data
+dataset = pd.read_csv(
```

Comment:
This should probably be in the main function and then passed into the plotting functions as needed